sources/packages/web/src/components/common/OverawardDetails.vue

```diff
@@ -1,112 +1,120 @@
 <template>
   <v-card class="mb-5">
```

Comment:
The idea was to no longer use `v-card` or `v-container`. Is it not taken care of by a common component?

Reply:
@sh16011993 for your ref. it must be taken care by body-header-container in this case.

Reply:
Fixed it.